.circleci/config.yml

```diff
@@ -2,19 +2,19 @@ version: 2.1
 
 executors:
   default:
-    docker:
-      - image: filecoin/rust:latest
-    working_directory: /mnt/crate
-    resource_class: 2xlarge
+    machine:
+      image: ubuntu-1604-cuda-10.1:201909-23
+    working_directory: ~/gpuci
+    resource_class: gpu.nvidia.medium
 
 restore-workspace: &restore-workspace
   attach_workspace:
-    at: /mnt
+    at: ~/
 
 restore-cache: &restore-cache
   restore_cache:
     keys:
-      - cargo-v1-{{ checksum "rust-toolchain" }}-{{ checksum "Cargo.toml" }}-{{ checksum "Cargo.lock" }}-{{ arch }}
+      - cargo-v0-{{ checksum "rust-toolchain" }}-{{ checksum "Cargo.toml" }}-{{ checksum "Cargo.lock" }}-{{ arch }}
```

Comment:
What does `v0`/`v1` mean here?

Reply:
it is just a manual switch to force a new cache on CI